web/src/beta/components/Resizable/index.tsx

```diff
@@ -33,24 +41,50 @@ const Resizable: React.FC<Props> = ({
   const BottomGutter = showBottomGutter ? <HorizontalGutter {...gutterProps} /> : null;
   const LeftGutter = showLeftGutter ? <VerticalGutter {...gutterProps} /> : null;
 
+  useEffect(() => {
+    if (size <= initialSize / 2) {
+      setMinimized(true);
+    } else {
+      setMinimized(false);
+    }
```

Comment:
done

Reply:
I think you misunderstood the above. The above is a suggested change, so I want this only
```
 if (!minimized && size <= initialSize / 2) {
      setMinimized(true);
    }
```
instead of your current changes. Seems you copy/pasted the above code, which includes both the old changes and my recommendation.

Reply:
sure , my bad , removed